doc/nrf/protocols/matter/getting_started/hw_requirements.rst

```diff
@@ -602,6 +626,111 @@ For example, for the ``nrf7002dk/nrf5340/cpuapp`` board target, the command is a
 
    west build -b nrf7002dk/nrf5340/cpuapp -t partition_manager_report
 
-.. note::
+Diagnostic logs RAM memory requirements
+=======================================
+
+:ref:`Diagnostic logs support<matter_configuration_diagnostic_logs>` requires the new configuration of the RAM memory to add three new retained partitions.
+The :ref:`matter_diagnostic_logs_snippet` adds the new RAM partitions which are configured as retained to keep the log data persistent and survive the device reboot.
+They also reduce the SRAM size of the Application core according to the size of the retained partition.
```

Comment:
```suggestion
The snippet also reduces the SRAM region available for the application by the size of the retained partitions.
```

Reply:
I want to suggest "reduces the amount of SRAM available for the application". Region is fairly vague in the context